pystarport/cosmoscli.py

```diff
@@ -661,47 +662,64 @@ def create_validator(
         commission_max_rate="0.2",
         min_self_delegation="1",
         event_query_tx=True,
+        sdk47_compact=True,
```

Comment:
@yihuang should we use legacy instead this flag?

Reply:
I'd prefer a separate method.